src-php/Adapters/CloudinaryAdapter.php

```diff
@@ -19,11 +19,19 @@ class CloudinaryAdapter extends CloudinaryBaseAdapter
      */
     public function writeStream($path, $resource, Config $config)
     {
-        $path = pathinfo($path)['filename'];
+        $pathinfo = pathinfo($path);
+        $folder = $pathinfo['dirname'];
+        $path = $pathinfo['filename'];
+
+        $options = ['public_id' => $path, 'resource_type' => 'auto'];
+
+        if ($folder != '.') {
```

Comment:
Does `pathinfo` always return a `.` for dirname if the file is in the cwd?

Reply:
As far as I remember, yes

Reply:
Looking at https://www.php.net/manual/en/function.dirname.php this could be a `.` `/` or `\`?

Reply:
When I was running it on mac (or Heroku) it was `.`. I don't have access to Windows machine to check what happens there